app/portainer/user-activity/activity-logs-view/activity-logs-datatable/activity-logs-datatable.html

```diff
@@ -0,0 +1,66 @@
+<div class="datatable datatable-empty">
+  <rd-widget>
+    <rd-widget-body classes="no-padding">
+      <datatable-titlebar title="Activity Logs" icon="fa-history"></datatable-titlebar>
+      <datatable-searchbar on-change="($ctrl.onChangeKeyword)" ng-model="$ctrl.keyword"></datatable-searchbar>
+      <div class="table-responsive">
+        <table class="table table-hover">
+          <thead>
+            <tr>
+              <th class="small-column">
+                <a ng-click="$ctrl.changeSort('Timestamp')">
+                  Time
+                  <datatable-sort-icon key="Timestamp" selected-sort-key="{{ $ctrl.sort.key }}" reverse-order="$ctrl.sort.desc"></datatable-sort-icon>
+                </a>
+              </th>
+              <th class="small-column">
+                <a ng-click="$ctrl.changeSort('Username')">
+                  User
+                  <datatable-sort-icon key="Username" selected-sort-key="{{ $ctrl.sort.key }}" reverse-order="$ctrl.sort.desc"></datatable-sort-icon>
+                </a>
+              </th>
+              <th class="small-column">
+                <a ng-click="$ctrl.changeSort('Context')">
+                  Environment
+                  <datatable-sort-icon key="Context" selected-sort-key="{{ $ctrl.sort.key }}" reverse-order="$ctrl.sort.desc"></datatable-sort-icon>
+                </a>
+              </th>
+
+              <th>
+                Action
+              </th>
+              <th>
+                Payload
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr dir-paginate-start="item in $ctrl.logs | itemsPerPage: $ctrl.limit" total-items="$ctrl.totalItems" current-page="$ctrl.currentPage">
+              <td></td>
```

Comment:
I'm not sure about this, as it makes the table with different code then EE, and we want the codebases to be similar

Reply:
we don't have the API service available for this, so I mocked the data. 
According to the design, it should be blank.